### DISCUSSION TITLE
Logs broken on macOS?

### DISCUSSION THREAD

**Discussion by johnholbrook** [elgatosf/elgato#41] [Help] [answered]
I'm running the latest macOS (13.4.1 at the time of writing) on an apple silicon Mac, and the latest version of the Stream Deck software (currently 6.3.0).

When I open `~/Library/StreamDeck/Logs`, the files in there all have modified dates of March 22 or earlier – looking through the [version history](https://help.elgato.com/hc/en-us/sections/5162671529357-Elgato-Stream-Deck-Software-Release-Notes) I suspect this is the date I updated to Stream Deck 6.1 which was released on March 13. It appears that no logs are being written, by the stream deck software itself or any installed plugins.

I verified that logs are still being written by the latest version of the stream deck software on Windows 10. 

Did 6.1 break writing logs on macOS, and if so is a fix for this in the works? Not having access to the logs makes working on my plugin significantly more difficult. My plugin has to a be a compiled binary for technical reasons so connecting to remote Chrome debugging is not an option.

Thanks in advance for any help!

**Comment by tiptronic** [comment, marked as answer]
Hi @johnholbrook 

please take a look at this path:

`~/Library/Logs/ElgatoStreamDeck`

or

`~/Library/Logs/StreamDeck `

for older versions.

**Reply by johnholbrook** [reply]
Ah, there they are – I was looking in `~/Library/Logs/StreamDeck` but they were in `ElgatoStreamDeck`.

FYI, the [developer documentation](https://docs.elgato.com/sdk/plugins/events-sent#logmessage) still lists the old path.

Thanks for your help!
